### PR TITLE
lathe recipes clean up

### PR DESCRIPTION
**Pull request by ErhardSteinhauer**
## About the PR
A bit of clean up for lathe recipes through parenting. Some recipes changed pricing but by a small margin, I don't think this warrants changelog entry.

## Why / Balance
Less lines

## Requirements
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
:cl:
- tweak: Some lathe recipes have changed material cost and print time.

### CODE REVIEW

File: Resources/Prototypes/_NF/Recipes/Lathes/weapons.yml [hunk 1]
```diff
@@ -1,48 +1,43 @@
...
-  category: Weapons
-  completetime: 5
   materials:
     Steel: 1200
     Glass: 800
     Silver: 850
 
 - type: latheRecipe
   id: WeaponRevolverArgenti
+  parent: BaseWeaponRecipe
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Recipes/Lathes/weapons.yml:13]: Did you want a change from 5 to 2 seconds?
```suggestion
  parent: BaseWeaponRecipeLong
```

File: Resources/Prototypes/_NF/Recipes/Lathes/weapons.yml [hunk 2]
```diff
@@ -1,48 +1,43 @@
 # Ranged weapons
 - type: latheRecipe
   id: WeaponLaserPistolNF
+  parent: BaseWeaponRecipe
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Recipes/Lathes/weapons.yml:4]: Did you want a change from 5 to 2 seconds?
```suggestion
  parent: BaseWeaponRecipeLong
```

File: Resources/Prototypes/_NF/Recipes/Lathes/weapons.yml [hunk 3]
```diff
@@ -1,48 +1,43 @@
...
   result: WeaponRevolverArgenti
-  category: Weapons
-  completetime: 5
   materials:
     Steel: 1500
     Plasteel: 500
 
 - type: latheRecipe
   id: WeaponPistolMk58NF
+  parent: BaseWeaponRecipe
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Recipes/Lathes/weapons.yml:21]: Did you want a change from 5 to 2 seconds?
```suggestion
  parent: BaseWeaponRecipeLong
```

File: Resources/Prototypes/_NF/Recipes/Lathes/weapons.yml [hunk 4]
```diff
@@ -94,73 +85,65 @@
...
   result: Katana
-  category: Weapons
-  completetime: 6
   materials:
     Plasteel: 500
     Plastic: 300
 
 - type: latheRecipe
   id: WeaponCrusher
+  parent: BaseWeaponRecipe
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Recipes/Lathes/weapons.yml:127]: Did you want a change from 5 to 2 seconds?
```suggestion
  parent: BaseWeaponRecipeLong
```

File: Resources/Prototypes/_NF/Recipes/Lathes/weapons.yml [hunk 5]
```diff
@@ -94,73 +85,65 @@
...
-  category: Weapons
-  completetime: 5
   materials:
     Steel: 1000
     Glass: 250
     Plastic: 100
 
 - type: latheRecipe
   id: WeaponCrusherDagger
+  parent: BaseWeaponRecipe
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Recipes/Lathes/weapons.yml:136]: Did you want a change from 5 to 2 seconds?  Unsure about this one, it's still a dagger, but it's also more useful.
```suggestion
  parent: BaseWeaponRecipeLong
```

File: Resources/Prototypes/_NF/Recipes/Lathes/storage.yml [hunk 6]
```diff
@@ -1,192 +1,148 @@
...
-  completetime: 1
-  applyMaterialDiscount: false
-  materials:
-    Cardboard: 100
 
 - type: latheRecipe
   id: NFFoodBoxSushi
+  parent: NFBaseBoxRecipe
   result: FoodBoxSushi
   name: lathe-recipe-NFFoodBoxSushi-name
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Recipes/Lathes/storage.yml:148]: Think this one should keep the plastic.
```suggestion
  name: lathe-recipe-NFFoodBoxSushi-name
  materials:
    Plastic: 100
```

**Review by whatston3 — COMMENTED** [review]
Largely looks fine, a few small questions about the parenting on some of the weapons, whether they should take 2 seconds or 5 to print.

Will test.

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**